Test/CoreSDK.Test/Shared/DataContracts/TelemetryContextTest.cs

```diff
@@ -26,10 +26,11 @@ public void TelemetryContextIsSealedToSupportCompilationAsWinmd()
         }
 
         [TestMethod]
+        [Ignore]
```

Comment:
Why `Ignore`? Add a comment or, if you never intend to bring it back, delete the test.